web/src/js/Menu.jsx

```diff
@@ -37,9 +38,9 @@ export default class Menu extends Component {
         </button>
         <div className="collapse navbar-collapse" id="navbarNavAltMarkup">
           <div className="navbar-nav">
-{this.menuItems.map((menuItem, index) =>
-            <a key={index} className={`nav-item nav-link ${window.location.pathname == menuItem.link ? "active" : ""} ${menuItem.requireLogin && !isLoggedIn() ? 'd-none' : ''}`} href={menuItem.link}>{menuItem.name}</a>
-          )}
+            {this.menuItems.filter(item => !item.requireLogin || item.requireLogin === isLoggedIn()).map((menuItem, index) => 
```

Comment:
suggestion: replace `===` with `&&` to clarify what's going on

```suggestion
            {this.menuItems.filter(item => !item.requireLogin || (item.requireLogin && isLoggedIn())).map((menuItem, index) => 
```